### PR TITLE
nuclear missile launcher, change sight, time value

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

**Comment by andy5995** [comment]
I think you can spot the changes pretty easy, despite all the white space removals.... I only changed the sight value and time value. What do you think?

**Comment by Jammyjamjamman** [comment]
Did you change the site to the attack range? I actually think the sight should be left alone for now, because too much vision adds "spying" power to a unit.

File: factions/enemies_and_allies/units/nuclear_missile_launcher_/nuclear_missile_launcher_.xml [hunk 1]
```diff
@@ -6,18 +6,18 @@
...
 		<max-hp value="2000" regeneration="0"/>
 		<max-ep value="300" regeneration="5"/>
-		<armor value="50"/>	
-		<armor-type value="metal"/>				
-		<sight value="15"/>
-		<time value="10"/>	
-		<multi-selection value="true"/>	
+		<armor value="50"/>
+		<armor-type value="metal"/>
+		<sight value="36"/>
```
Comment (andy5995) [inline factions/enemies_and_allies/units/nuclear_missile_launcher_/nuclear_missile_launcher_.xml]:
```suggestion
		<sight value="15"/>

```

**Review by Jammyjamjamman — APPROVED** [review]
All look sensible.

**Pull request merged**